profiles/fscloud/main.tf

```diff
@@ -22,3 +22,12 @@ module "postgresql_db" {
   member_cpu_count          = var.member_cpu_count
   members                   = var.members
 }
+
+# Create IAM Authorization Policies to allow COS to access kms for the encryption key
+resource "ibm_iam_authorization_policy" "primary_kms_policy" {
+  source_service_name         = "databases-for-postgresql"
+  source_resource_instance_id = module.postgresql_db.guid
+  target_service_name         = "hs-crypto"
+  target_resource_instance_id = var.existing_hpcs_instance_guid
+  roles                       = ["Reader"]
+}
```

Comment:
Updates needed in "profiles/fscloud/variables.tf"
- `key_protect_key_crn` needs to be renamaed to `hpcs_key_crn` and descrption updated 
- `backup_encryption_key_crn` variable descrption needs to be updated to specify hyper protect, not key protect
- `region` variable description needs to be updated as it mentions BYOK (which is key protect only) and fscloud mandates KYOK (Hyper Protect)
- `resource_tags` are only applied to the Postgres instance, so maybe ensure thats clear in the variable descrption 
- `allowlist` variable should probably be removed. CBRs should be used instead.
- Should we passing values in the variable `configuration`? Are any of these configuration required for FSCloud? Has an SCC scan been done against an instance provisioned using this config?

Reply:
removed `allowlist` and `configuration`. There was no `configuration` set in the profile or the example consuming it.  I will run SCC scan against the instance and will verify if any `configuration` might be needed.